lldb/test/Shell/lit.cfg.py

```diff
@@ -179,3 +180,14 @@ def calculate_arch_features(arch_string):
 
 if "LD_PRELOAD" in os.environ:
     config.available_features.add("ld_preload-present")
+
+# Determine if a specific version of Xcode's linker contains a bug. We want to
+# skip affected tests if they contain this bug.
+try:
+    raw_version_details = subprocess.check_output(("xcrun", "ld", "-version_details"))
+    version_details = json.loads(raw_version_details)
+    version = version_details.get("version", "0")
+    if 1000 <= float(version) <= 1109:
```

Comment:
float will be insufficient, this will need to handle versions such as `1100.1.1`